biotrainer/config/embedding_options.py

```diff
@@ -144,5 +145,60 @@ def allow_download(self) -> bool:
         return True
 
 
+class DimensionReduction(EmbeddingOption, ConfigOption):
+
+    @classproperty
+    def name(self) -> str:
+        return "dimension_reduction_method"
+
+    @property
+    def default_value(self) -> Union[str, int, float, bool, Any]:
+        return ""
+
+    @classproperty
+    def allow_multiple_values(self) -> bool:
+        return False
+
+    @property
+    def possible_values(self) -> List[Any]:
+        return ["umap", "tsne"]
+
+    @classproperty
+    def allowed_protocols(self) -> List[Protocol]:
+        return [Protocol.sequence_to_class, Protocol.sequence_to_value]
```

Comment:
You could use Protocol.using_per_sequence_embeddings() here instead.

Reply:
I copied this part from the Interaction class in general_options.py, should I change it there too?